app/controllers/candidate_interface/degrees/institution_controller.rb

```diff
@@ -4,7 +4,7 @@ class InstitutionController < BaseController
       before_action :set_institution_names, :set_countries
 
       def new
-        @degree_institution_form = DegreeInstitutionForm.new(degree: current_degree)
+        @degree_institution_form = DegreeInstitutionForm.new(degree: current_degree).assign_form_values
```

Comment:
I think there is more to do in this controller. When I tested this I was bumped to the review page after filling out the institution rather than moving to the grade form(s). 

I think this needs updating:
https://github.com/DFE-Digital/apply-for-teacher-training/blob/ac695ec8db4b7e7eb9b2ead1d4e7427eab2e4d1a/app/controllers/candidate_interface/degrees/institution_controller.rb#L17

Reply:
hmm that's interesting. I'm seeing the flow as expected. If you've got 5 mins later can you show me please 🙏

Reply:
As discussed my mistake, have retested and all the backlinks work as well as the normal flow